_locales/fr/messages.json

```diff
@@ -61,69 +61,69 @@
         "message": "Minutes"
     },
     "Secs": {
-        "message": "Seconds"
+        "message": "Secondes"
     },
     "Hide": {
-        "message": "Ne Montre Jaimais"
+        "message": "Toujours cacher"
     },
     "hitGoBack": {
-        "message": "Clique retourne pour si vous avez manqué parti."
+        "message": "Revenir au début du segment"
     },
     "unskip": {
-        "message": "Retourne"
+        "message": "Ne pas sauter"
```

Comment:
I don't know about this one, this is for the action of going back to the beginning after the skip.

Reply:
In the sense of "Oops I wanted to see that" this could be "Revenir en arrière" (go back) but it feels a bit long. Since the skip/jump is always forward by default, "un-jump" is my understanding but "Sauter en arrière" is more like "jump backward" than "jump back". So I chose a short one with "Ne pas sauter" which indicates more the "I didn't/don't want that" than "do this now".

Reply:
Is `retourne` not a good word for it?

Reply:
"Retourne" is not a noun, nor a verb that can be displayed as is, this would be "Retournez" (go back) but without anything else we don't know where we go back. "Retourne" is also "flipping it over" so "Revenir" is better, more temporal. Back is "en arrière" so "Revenir en arrière" is "Go back". Again this indicates what the button does, but not the context that the result of the previous action might not have been what we expected, but it's a very slim line and I think people would understand both.